### ISSUE TITLE
Rolling back to 9 was a hassle

### ISSUE THREAD

**Issue by ljtelllef** [ubports/ubports-installer#2262]
**UBports Installer `0.8.9-beta` (exe)**
Environment: `Microsoft Windows 10 Pro 10.0.19042 win32 10.0.19042 x64 19042 0.0 NodeJS v12.18.3`
Device: enchilada
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: *N/A*
Log: *N/A*

Buy a OP6 that has 11.1.1.1 on it, and try to roll back to 9.  Major hassle and learning experience for a nOOb.  Some directions or an installer would have been helpful.

**Comment by mellowdd** [comment]
Did it work?

**Comment by ljtelllef** [comment]
Yes, but the app store appears to be a static image.   It was one of 3 apps
that updated immediately.

**Comment by ljtelllef** [comment]
I worked before the update (I was hoping to find a plex app).  If I can get
the basics working,  I have a pixel 3 to setup.

I also have 3 one plus one's that I am selling, if you know anyone
interested (I can't get find coverage in Dallas).

Thanks for checking.
